discovery/syncer.go

```diff
@@ -327,8 +343,17 @@ func (g *GossipSyncer) Start() {
 	g.started.Do(func() {
 		log.Debugf("Starting GossipSyncer(%x)", g.cfg.peerPub[:])
 
-		g.wg.Add(1)
-		go g.channelGraphSyncer()
+		// TODO(conner): only spawn channelGraphSyncer if remote
```

Comment:
Won't these two new variables handle this? Or is this referring to handling it at the server/p2p level?

Reply:
it's possible to do so with the new variables, but this TODO is referring to hooking them up to the negotiated feature bits